languages.toml

```diff
@@ -19,6 +19,11 @@ config = """
 
 language-server = { command = "rust-analyzer" }
 indent = { tab-width = 4, unit = "    " }
+debug-adapter = { name = "lldb", transport = "tcp", command = "lldb-vscode", args = [], port-arg = "-p {}" }
+
+[[language.debug-configs]]
+console = "internalConsole"
+program = "target/debug/rustdebug"
```

Comment:
This should be template for user. Start command will be typable (not keybinding) and have the following format:
`:debug-run <name> <param1> <param2> ...` where `<name>` is for example `test` or anything else being the name of launch config (they must have names later) and params get interpolated into template

Reply:
I think you also need to allow for specifying launch/attach here?

Reply:
Yes. I'm going to think a bit on types. I want it like this (JSON, but will be done as TOML):
```json
"debugger": {
  "name": "lldb",
  ...
  "templates": [
    {
      "name": "execute",
      "request": "launch",
      "args": { "program": "{}", ... }
    }
  ]
}
```